src/sass/index.scss

```diff
@@ -1,51 +1,22 @@
-* {
-  box-sizing: border-box;
-  padding: 0;
-  margin: 0;
-  font-family: Poppins, "Courier New", Courier, monospace;
-}
-
-.list {
-  margin: 10px 0 0 10px;
-  width: 90%;
-  list-style: none;
-  box-shadow: 5px 2px 10px 2px rgba(0, 0, 0, 0.1);
-
-  @media screen and (min-width: 992px) {
-    width: 600px;
-  }
-
-  .list-item {
-    width: 100%;
-    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
-    padding: 1rem;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-
-    .wrapper {
-      display: flex;
-      align-items: center;
-      gap: 10px;
-    }
-
-    &__button-wrapper {
-      padding: 0;
-
-      .btn {
-        padding: 1rem;
-        border: 0;
-        background-color: rgba(0, 0, 0, 0.1);
-        color: rgba(0, 0, 0, 0.6);
-        width: 100%;
-        height: 100%;
-      }
-    }
-
-    .add-item {
-      border: 0;
-      width: 100%;
-      height: 100%;
-    }
-  }
-}
+/* -------------------------------------------------------------------------- */
+
+/*                               //IMPORT CUSTOM                              */
+
+/* -------------------------------------------------------------------------- */
+@import "./custom";
+
+/* -------------------------------------------------------------------------- */
+
+/*                             //IMPORT COMPONENTS                            */
+
+/* -------------------------------------------------------------------------- */
+@import "./components/mixins";
```

Comment:
- [ ] It seems like there are compiling problems: I'm getting the following error on my local environment. Can you make sure to include `./components/mixins` in the repository or remove it from this file if it is empty?

![image](https://user-images.githubusercontent.com/79114233/149259807-d7fbf25e-1060-432d-8cd5-d9b2a76c978d.png)

Reply:
I removed the line 13 assuming mixins is an empty file that didn't get pushed to GitHub, but then I get this error: 
![image](https://user-images.githubusercontent.com/79114233/149260015-6d80d3e5-8357-42c5-81e4-d713171818f1.png)

Make sure to fix all of these issues so the next reviewer can perform a review without compiling errors. I recommend you that if extra steps need to run the project locally you include these on the README file.